### PR TITLE
Add "select" function to "MySQLHelper" class

### PR DESCRIPTION
**Pull request by tytus-kurek**
This patchset adds "select" function to "MySQLHelper" class which
can be used to execute SQL select query and return the results.

Closes-Bug: 1775182

### CODE REVIEW

File: charmhelpers/contrib/database/mysql.py [hunk 1]
```diff
@@ -163,6 +163,17 @@ def execute(self, sql):
         finally:
             cursor.close()
 
+    def select(self, sql):
+        """Execute arbitrary SQL select query against the database
+         and return the results."""
```
Comment (ajkavanagh) [inline charmhelpers/contrib/database/mysql.py]: Please could we do full doctring comments.  In this case something like:

```python
"""Execute ...

    :param sql: the sql string to execute.
    :type sql: str
    :returns: list of strings? <-- what does it return?
    :rtype: [str]
    :raises: <-- which Execption classes can it raise?
"""

**Comment by ajkavanagh** [comment]
@tytus-kurek unfortunately you've got a pep8 failure which is just white space formatting.

**Comment by tytus-kurek** [comment]
@ajkavanagh sorry for the mess around it :).

**Comment by ajkavanagh** [comment]
Thanks @tytus-kurek for the patch and changes.

**Pull request merged**